environments/prod/mailrelay-platform-hmcts-internal.yml

```diff
@@ -0,0 +1,39 @@
+name: "mailrelay.platform.hmcts.internal"
```

Comment:
why are we using an invalid domain? and also one we don't control?

if you want a separate domain something like:

internal.platform.hmcts.net

or 

platform-int.hmcts.net (would require getting control of)

but before you do anything can you explain the rationale?

Reply:
Hey, I totally agree with you here and I'm updating it. We did talk about this in refinement and I have mailrelay.internal.hmcts.net in my notes - possibly the wrong one ended up on the ticket.

Reply:
Yeah good idea. All my fault :-) Thanks Tim